vignettes/cookbook_interpretation.Rmd

```diff
@@ -14,59 +14,164 @@ knitr::opts_chunk$set(
+
+The primary column/variable of interest in the output is the `emission_factor_value` variable. What the `emission_factor_value` variable represents depends on the values in all of the other columns, and in particular the `emission_factor_metric` column. If the value in the `emission_factor_metric` column on the same row is `"projected"`, then the value in the `emission_factor_value` variable will represent the physical emission intensity level of your portfolio based on the ABCD data for that sector/year/region/scenario/etc. If the value in the `emission_factor_metric` column on the same row begins with `"target_"`, e.g. `"target_cps"`, then the value in the `emission_factor_value` variable will represent the target level of physical emission intensity for that sector/year/region/scenario/etc. based on the ABCD start year production value of your portfolio modified to be relative to the scenario data for the given year. If the value in the `emission_factor_metric` column on the same row is `"corporate_economy"`, then the value in the `emission_factor_value` variable will represent [the total physical emission intensity levels of the entire sector adjusted by the start year physical emission intensity level of your portfolio, which can be used aa a benchmark to compare to your portfolio's production levels]. If the value in the `emission_factor_metric` column on the same row starts with `"adjusted_"`, then the value in the `emission_factor_value` variable will represent [?].
+
+If you use the `by_company = TRUE` option of the `target_sda()` function, most of what's said above is true except relative to the company listed in that row in the `name_abcd` column.
+
+For detailed information about the table output of the `target_sda()` function and its columns, see the [data dictionary](data_outputs.html#target_sda-output).
+
+#### Interpretation & Intuition
+
+[2 or 3 sentences about interpretation]
+
+[2 or 3 sentences about intuition]
+
+### Emission Intensity Plot
+
+The emission intensity metric measures the average CO^2^ intensity of the portfolio in the steel and cement sector. This emission intensity is given as CO^2^/economic unit of output (for example, CO^2^/per ton of steel produced). This is then compared to an emission intensity reference point set by a climate scenario.
+
+While this is not the main metric of choice for the largest sectors tackled in this methodology, the emission intensity of the activities financed by the portfolio is nonetheless the first metric in sectors for which no clear technology pathways have been set out (namely, steel and cement). Put differently, for these sectors, no zero-carbon alternative yet exists. As such, it is not possible to use the technology mix metric or the volume production volume trajectory metric to measure alignment. However, it is still imperative to steer capital in a way that aims to decrease carbon emissions in these sectors – hence the emission intensity metric is used.
```

Comment:
```suggestion
The emission intensity of the activities financed by the portfolio is the metric of choice in sectors for which no clear technology pathways have been set out. Put differently, for these sectors, no zero-carbon alternative yet exists. As such, it is not possible to use the technology mix metric or the volume production volume trajectory metric to measure alignment. However, it is still imperative to steer capital in a way that aims to decrease carbon emissions in these sectors – hence the emission intensity metric is used.
```

Reply:
done in https://github.com/RMI-PACTA/pacta.loanbook/pull/129/commits/da4a94787ac71ea4173f10fa49ad7227686ff774